lib/models/users.js

```diff
@@ -1,12 +1,26 @@
-const { getCollection } = require("../database");
+const mongoose = require("mongoose");
 
-async function readUsers() {
-  const userCollection = await getCollection("users");
-  const users = await userCollection.find({}).toArray();
+const UserSchema = mongoose.Schema({
+  name: {
+    type: String,
+    required: true,
+    trim: true,
+  },
+  email: {
+    type: String,
+    required: true,
+    trim: true,
+    unique: true,
+  },
+  password: {
```

Comment:
I also would suggest to use a `min-` and `max-length` as you already did it in your `ArticlesModel`

Reply:
I do that in the authController!